.github/scripts/export_pytorch_labels.py

```diff
@@ -14,12 +14,25 @@
 import json
 
 from label_utils import gh_get_labels
+from argparse import ArgumentParser
+from typing import Any
+
+
+def parse_args() -> Any:
+    from argparse import ArgumentParser
+    parser = ArgumentParser("Export PR labels")
+    parser.add_argument("org", type=str)
```

Comment:
Nit: You can also set `required=True` here I think.  The script won't run otherwise anyway

Reply:
These ones are required by default, since they're positional arguments.  If you leave them out you get an error